### ISSUE TITLE
Previous results: A summary table of reported results from previous years

### ISSUE THREAD

**Issue by hammerdirt-analyst** [hammerdirt-analyst/qualite-deau#10]
Bonjour,

Nous allons revoir l'analyse d'une année sur l'autre.  Existe-t-il un tableau récapitulatif des valeurs rapportées les années précédentes ?

Nous pourrions également inclure les documents utilisés pour communiquer les résultats aux partenaires. Nous pouvons créer un tableau récapitulatif à partir de ces documents.

Je vous remercie de votre attention. 

============= English ==========================================

Hello,

We will be revisiting the analysis year over year.  Is there a summary table that has the reported values from the previous years?

As a second best alternative we could include the documents that were used to report results to stakeholders. We can build a summary table from that.

Thank you

**Comment by rachelaronoff** [comment]
pour le moment, on n'a pas un tableau des années précédentes. prête de le construire (en attendante savoir lesquelles valeurs exactement...)  Merci!

**Comment by hammerdirt-analyst** [comment]
J'ai cru comprendre que les résultats étaient calculés pour chaque année d'échantillonnage. Peu importe ce qui a été rapporté tant que nous savons ce qu'il en est.

Les résultats ont-ils été calculés ou non ? Si c'est le cas, nous aimerions inclure les résultats précédents dans ce repo.

===============

It was my understanding that the results were calculated for each sampling year. It does not matter what was reported as long as we know what it was.

Were the results calculated or not? If so we would like to include the previous findings in this repo.

**Comment by rachelaronoff** [comment]
of course.  These data were indeed calculated, and should be included also here.  I think the tabulated results (site by site, or averaged around the bay?) could be easily put together.  (Just gave Zenodo archive info link in the issue about 24/48h, so won't put it here too.) 
For the 2022 data, we have done some site by site analyses - [for instance, presented here](https://wiki.hackuarium.ch/images/9/97/Mcbp_prelim.pdf), in the Hackuarium wiki..

**Comment by hammerdirt-analyst** [comment]
Hello,

Please furnish the data in a fashion that makes sense to you. We are not correcting or criticiszing previous work. This is for the sake of completeness.

We can leave this issue open and add the historical results at  a later date.

Thanks

**Comment by rachelaronoff** [comment]
ok. thx

**Comment by rachelaronoff** [comment]
here is an example of the graph averaging all three sites around the bay (was in 20Min in 2022!) based just on the data in the 2021 article. 
![IMG_5831](https://github.com/hammerdirt-analyst/qualite-deau/assets/48484650/5d97fbee-13dd-4f10-b2ee-113bffbb9971)
